ui/src/transaction.ts

```diff
@@ -542,55 +536,43 @@ export const summaryText = (counts: SummaryCounts): string => {
 };
 
 interface CostSummary {
-  depositRad: currency.Rad;
-  depositUsd: currency.Usd;
-  feeRad: currency.Rad;
-  feeUsd: currency.Usd;
-  totalRad: currency.Rad;
-  totalUsd: currency.Usd;
+  registrationFee?: FeeAmount;
+  txFee: FeeAmount;
+  total: FeeAmount;
 }
 
-export const costSummary = (
-  messageType: MessageType,
-  fee: currency.MicroRad,
-  deposits: Deposits
-): CostSummary => {
-  let deposit = 0;
-
-  switch (messageType) {
-    case MessageType.OrgRegistration:
-      deposit = deposits.orgRegistration;
-      break;
-    case MessageType.MemberRegistration:
-      deposit = deposits.memberRegistration;
-      break;
-    case MessageType.ProjectRegistration:
-      deposit = deposits.projectRegistration;
-      break;
-    case MessageType.UserRegistration:
-      deposit = deposits.userRegistration;
-      break;
-    default:
-      throw `MessageType: ${messageType} not implemented`;
-      break;
-  }
+const paysRegistrationFee = (messageType: MessageType): boolean => {
+  return (
+    messageType === MessageType.OrgRegistration ||
+    messageType === MessageType.UserRegistration
+  );
+};
```

Comment:
`paysRegistrationFee` should be decided on the values of `session.registrationFee`.

Reply:
forgot to delete that, it's no longer used!